backend/utils/enums.js

```diff
@@ -0,0 +1,19 @@
+const PrintfulWebhookEvents = {
```

Comment:
nit: do you mind adding a comment to remind developers that if they make any change to those definitions, they also need to add a DB migration file to add/delete whatever enum value they are adding/deleting?